pkgs/development/libraries/libtomcrypt/default.nix

```diff
@@ -21,20 +21,29 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  nativeBuildInputs = [ libtool libtommath ];
+  nativeBuildInputs = [ libtool ];
+  buildInputs = [ libtommath ];
+
+  outputs = [ "out" "dev" ];
 
   postPatch = ''
     substituteInPlace makefile.shared --replace "LIBTOOL:=glibtool" "LIBTOOL:=libtool"
   '';
 
+  env.CONFIG_CFLAGS = "-DUSE_LTM -DLTM_DESC -DLTC_PTHREAD";
```

Comment:
`USE_LTM` is not really required in the pkg-config file, since it's an internal option of the ltc tests.

Reply:
Oh, I didn't realize that `USE_LTM` was internal. I just knew that the brute force approach of propagating all the flags worked. :upside_down_face:

I just force pushed and split `CONFIG_CFLAGS` into `BUILD_CFLAGS` and `USER_CFLAGS`.